www/docs/detailed-usage.rst

```diff
@@ -21,18 +36,14 @@ Get and build rovercode as usual:
     $ git clone --recursive https://github.com/aninternetof/rovercode.git && cd rovercode
     $ sudo docker build -t rovercode .
 
-Open `app.py` and comment/uncomment the `ROVERCODE_WEB_REG_URL` lines to select
-`rovercodeweb:8000`. You want it to end up like this:
-
-.. code-block:: python
+Set the url of the rovercode-web target to `http://rovercodeweb:8000`. You will
+see in the next step that this is the hostname that we assign to our local
+rovercode-web container.
 
-    ''' Default. Use with rovercode-web running at rovercode.com '''
-    #ROVERCODE_WEB_REG_URL = "https://rovercode.com/mission-control/rovers/"
-    ''' Uncomment this line to use with rovercode-web running at beta.rovercode.com '''
-    #ROVERCODE_WEB_REG_URL = "https://beta.rovercode.com/mission-control/rovers/"
-    ''' Uncomment this line to use with a local rovercodeweb docker container '''
-    ROVERCODE_WEB_REG_URL = "http://rovercodeweb:8000/mission-control/rovers/"
+.. code-block:: bash
 
+    # first, navigate to the rovercode root diretory (same level as the Dockerfile), then
+    $ echo ROVERCODE_WEB_URL=http://rovercodeweb:8000/ >> .env
```

Comment:
>> will append the setting as a new line.  If the statement is run more than once it will create additional lines of the same setting, possibly with different values.  Is there ever anything else in this file?  Maybe it would be better to use > to force the file to be recreated, or just create the file by default with the setting.